src/Movie-Card/Movie-Card.css

```diff
@@ -0,0 +1,75 @@
+.movie-card {
+  height: 90%;
+  width: auto;
+  padding: 0;
+  margin: 20px;
+  border-radius: 5px;
+  position: relative;
+  padding-left: 10px;
+  align-content: center;
+  font-size: 0.8em;
+  font-family: sans-serif;
+  color: whitesmoke;
+  font-kerning: normal;
+  text-shadow: 0ch;
+  outline-color: black;
+}
+.movie-card:hover {
+  cursor: pointer;
+}
+.movie-card:hover .hidden {
+  opacity: 1;
```

Comment:
```suggestion
  cursor: pointer;
  opacity: 1;
```

Reply:
Adding a cursor property to all the elements seems to get rid of that bug where you where not getting the pointer